forge/src/main/java/com/wynntils/forge/mixins/ForgeIngameGuiMixin.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.forge.mixins;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.mc.EventFactory;
+import net.minecraft.client.Minecraft;
+import net.minecraftforge.client.gui.ForgeIngameGui;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(ForgeIngameGui.class)
+public abstract class ForgeIngameGuiMixin {
+
+    // TODO: This is kinda hacky, but we can't shadow minecraft here
+    //       so we have to use the instance.
+
+    @Inject(method = "render", at = @At(value = "HEAD"))
+    public void onRenderGuiPre(PoseStack poseStack, float partialTick, CallbackInfo ci) {
+        EventFactory.onRenderGuiPre(
```

Comment:
I think you should add a comment to the "common" injection that it does not apply to Forge, and reference this class.